README.md

```diff
@@ -31,13 +31,11 @@ $ composer update
 $ php artisan vendor:publish --provider="Sportmonks\SoccerAPI\SoccerAPIServiceProvider"
 ```
 
-**4-** Review the configuration file and add your token in your .env file: `'api_token' => env('API_TOKEN')`
+**4-** Add your API token in the .env file to `SPORTMONKS_API_TOKEN`
 
-***If you don't have a API-token, you can get a free one [here](https://www.sportmonks.com/register) This should be enough to experiment with the code.***
+Add `env('SPORTMONKS_API_TOKEN')` to `api_token` in `config/soccerapi.php`.
```

Comment:
This is in the new soccerapi-config file

Reply:
I'm not sure what you mean. Are you asking me to change `config/soccerapi.php` to `config/soccerapi-config.php`?

Reply:
no, the soccerapi.php that is shipped with the laravel-sportmonks-soccer package already contains this line, if using version dev-master

Reply:
I see you're talking about this? https://github.com/kirill-latish/laravel-sportmonks-soccer/blob/master/src/config/config.php

Nice so I'll just remove that from the README then.

Reply:
Exactly! I asked the developer of the package to release a new version so we don't need to update the dev-master instead of ^v2.0 in your `package.json`